src/components/BillCategoryTag/BillCategoryTag.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+	import { Tag } from 'carbon-components-svelte';
+
+	export let isLarge = false;
+	let label = 'ขนส่งสาธารณะ';
```

Comment:
The label should be a "prop" without a default value. Every category has the same priority and adding a default value makes the prop become optional which we don't want that. Then you can pass `label="ขนส่งสาธารณะ"` as a sample in the story.